### ISSUE TITLE
Learning Mode - Style tweaks for individual Learning Mode templates

### ISSUE THREAD

**Issue by donnapep** [Automattic/course#47]
Dependent on #48.

Note that there's a [separate issue](https://github.com/Automattic/sensei-theme/issues/46) for adjusting common styles across all Learning Mode templates.

Once https://github.com/Automattic/sensei-theme/issues/48 is implemented, some of these styling tweaks may no longer be necessary.

## Video Template

Update the Video Learning Mode template to match the [design](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020082%3A4588).

In particular, the course progress bar text needs to be in uppercase, and the font family and font size updated. The progress bar itself also needs to be taller and the border radius adjusted.

![Screen Shot 2022-10-20 at 1 57 39 PM](https://user-images.githubusercontent.com/1190420/197023293-fa63ca8f-4b41-4973-bd84-8c743927e559.jpg)

## Default & Large Video Templates

Update the Default and Large Video templates to match the [design](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020083%3A5966).

In particular, the font family, font size and color of the "2 of 10 lessons complete (25%)" text in the header needs to be updated. Also, the background color of the progress bar needs to be tweaked (the fill color is correct).

![Screen Shot 2022-10-20 at 2 06 00 PM](https://user-images.githubusercontent.com/1190420/197024871-6eaff539-0068-40ed-84c5-ae98b1f631d4.jpg)

## Modern & Video & Large Video Templates

Update the [Modern](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20021361%3A11212), [Video](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020082%3A4588) and [Large Video](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020093%3A1715) templates to match the design.

In particular, the border radius around the video + course outline container in the Video template, and the border radius around the course outline container in the Modern and Large Video templates need to be adjusted.

![Screen Shot 2022-10-20 at 2 12 01 PM](https://user-images.githubusercontent.com/1190420/197026016-fb426967-abed-486a-ba0a-e3d837ee0f57.jpg)

![Screen Shot 2022-10-20 at 2 14 06 PM](https://user-images.githubusercontent.com/1190420/197026472-2cbefe25-c54c-40e7-b53f-6fc47ca74b47.jpg)

#### Scenario: Default template
**Given** The Sensei theme is activated
**And** Learning Mode is enabled for all courses
**And** the Default Learning Mode template is activated
**When** I view a lesson
**Then** the header text and progress bar background color look similar to the design.

#### Scenario: Modern template
**Given** The Sensei theme is activated
**And** Learning Mode is enabled for all courses
**And** the Modern Learning Mode template is activated
**When** I view a lesson
**Then** the border radius around the course outline container looks similar to the design.

#### Scenario: Video template
**Given** The Sensei theme is activated
**And** Learning Mode is enabled for all courses
**And** the Video Learning Mode template is activated
**When** I add the Featured Video block to a lesson
**And** I view that lesson on the frontend
**Then** the border radius around the video + course outline container looks similar to the design
**And** the course progress bar text is in uppercase and has the correct font family and size
**And** the progress bar has the correct height and border radius.

#### Scenario: Large Video template
**Given** The Sensei theme is activated
**And** Learning Mode is enabled for all courses
**And** the Large Video Learning Mode template is activated
**When** I add the Featured Video block to a lesson
**And** I view that lesson on the frontend
**Then** the header text and progress bar background color look similar to the design
**And** the border radius around the course outline container looks similar to the design.
